ledger-api/grpc-definitions/com/daml/ledger/api/v1/admin/party_management_service.proto

```diff
@@ -200,4 +200,9 @@ message PartyDetails {
   // Optional,
   // Modifiable
   ObjectMeta local_metadata = 4;
+
+  // The id of the ``Identity Provider Config``
+  // If not provided, it would be assumed a default identity provider config.
+  // Optional
+  string identity_provider_config_id = 5;
```

Comment:
1. What are the implications of a party having this value assigned?
2. Is this value immutable?
3. Is this value stored exclusively on the local participant-node? If so, will there be a way to assign it on participants nodes other that one that this party has been originally allocated via a LAPI call?

`identity_provider_config_id` is designed to be immutable, correct?

Reply:
correct, `identity_provider_config_id` is designed to be immutable

I actually thought this would be required to list known parties which are local, but it seems that this is not actually required.